pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "blurry-cli"
-version = "0.6.2"
+version = "0.6.2.2"
```

I can bump the version on the next release

```suggestion
version = "0.6.2"
```